node/src/Pos/Client/Txp/Util.hs

```diff
@@ -298,9 +310,18 @@ makeLenses ''InputPickerState
 
 type InputPicker = StateT InputPickerState (Either TxError)
 
+-- | Most simple way of picking inputs:
+--
+-- * prefer ones which are not used in this slot
+--
+-- * take ones which have more coins first (in order to decrease number of
+-- picked inputs and thus decrease fee)
 plainInputPicker :: PendingAddresses -> InputPickingWay
-plainInputPicker (PendingAddresses pendingAddrs) utxo _outputs moneyToSpent =
-    evalStateT (pickInputs []) (InputPickerState moneyToSpent sortedUnspent)
+plainInputPicker (PendingAddresses pendingAddrs) utxo =
```

Comment:
I think one thing that might make this clearer is rewriting this to use `let-` binding before the lambda: e.g. 
```
let
     sortedUnspent = confirmed ++ unconfirmed
 	 
     (confirmed, unconfirmed) =	...
in \_outputs moneyToSpend  -> ...
```

This makes it more obvious that the purpose here is to share the evaluation of these things.

Reply:
Right, this makes sence.